src/CoreBundle/Controller/ChameleonFrontendController.php

```diff
@@ -36,17 +36,15 @@ class ChameleonFrontendController extends ChameleonController
      */
     private $configToLessMapper;
 
-    /**
-     * @param RequestStack                       $requestStack
-     * @param EventDispatcherInterface           $eventDispatcher
-     * @param PortalDomainServiceInterface       $portalDomainService
-     * @param TModuleLoader                      $moduleLoader
-     * @param IViewPathManager|null              $viewPathManager
-     * @param ContainerInterface                 $container
-     * @param TPkgViewRendererConfigToLessMapper $configToLessMapper
-     */
-    public function __construct(RequestStack $requestStack, EventDispatcherInterface $eventDispatcher, PortalDomainServiceInterface $portalDomainService, TModuleLoader $moduleLoader, $viewPathManager, ContainerInterface $container, TPkgViewRendererConfigToLessMapper $configToLessMapper)
```

Comment:
Just a note that the Symfony coding standards require to declare all method arguments in a single line (https://symfony.com/doc/current/contributing/code/standards.html under "Structure"). I'm not the biggest fan of that rule, so it's OK for me to wrap lines. Maybe we enforce the rule in the future, though.